schema/configuration_endpoint.json

```diff
@@ -21,6 +21,10 @@
     "hostname": {
       "type": "string"
     },
+    "error": {
```

Comment:
What's this?

Reply:
It's something for capturing errors during parsing/handling -- I updated the description slightly to reflect this.